lib/JobProperties.pm

```diff
@@ -190,11 +186,6 @@ sub _set_defaults
 	$self->set_file('shuf', $shuf) if ((defined $shuf) and (-e $shuf));
 	
 	$self->set_file('nucmer', $nucmer) if ((defined $nucmer) and (-e $nucmer));
```

Comment:
I think this is the last line that should be removed, since we don't use nucmer anymore.